src/projects/TodoList/index.js

```diff
@@ -132,6 +155,9 @@ class TodoList extends React.Component {
           whilstEditing={(e) => {
             this.whilstEditing(i, e.target.value);
           }}
+          deleteCategory={(e, i) => {
+            this.deleteCategory(i);
+          }}
```

Comment:
Ignore this. Forgot to delete this code when I realised I'd put it in the wrong place.